parlai/chat_service/README.md

```diff
@@ -0,0 +1,65 @@
+# Setting up ParlAI for a custom chat-service
+This is an instruction manual to be used as reference on how to configure ParlAI for an arbitrary chat service.
+
+## Message Format
+To maintain consistency we are trying to enforce a deterministic message format throughout this task. If a particular chat service doesn't adhere to this format, one must use ```restructure_message()``` to adapt the messages to this format before the messages being used. The message format is defined below:
+```config
+{
+  mid: # ID of this message
+  recipient: {id: #id of message recipient}
+  sender: {id: #id of message sender}
+  text: # text of the message
+  attachment: # attachment of the message
+}
+```
+### TBD
+- messaging_type: # TODO [RESPONSE, UPLOAD]
+- quickreplies: # TODO
+- persona_id: # TODO
+- name: # TODO
+- profile_picture_url: # TODO
+
+
+## Config
+Below is the standard config format and hierarchy to be followed across all chat services:
+
+```config
```

Comment:
i'm not sure this config setup is correct, see https://github.com/facebookresearch/ParlAI/blob/master/parlai/chat_service/tasks/overworld_demo/config.yml for an example (i.e., under `tasks` is another key->dict mapping from task name to onboard_world, etc.